File: CS/Q556797/Controllers/HomeController.cs
using MVCxGridViewDataBinding.Models;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.Mvc;

namespace Q556797.Controllers
{
    public class HomeController : Controller
    {
        //
        // GET: /Home/

        public ActionResult Index()
        {
            return View();
        }
        public ActionResult GridPartial() {
            var model = MyModel.GetModelList();
            return PartialView(model);
        } 
    }
}

File: VB/Q556797/Controllers/HomeController.vb
Imports Microsoft.VisualBasic
Imports MVCxGridViewDataBinding.Models
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Web
Imports System.Web.Mvc

Namespace Q556797.Controllers
	Public Class HomeController
		Inherits Controller
		'
		' GET: /Home/

		Public Function Index() As ActionResult
			Return View()
		End Function
		Public Function GridPartial() As ActionResult
			Dim model = MyModel.GetModelList()
			Return PartialView(model)
		End Function
	End Class
End Namespace
